<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskQueueWindow</class>
 <widget class="QMainWindow" name="TaskQueueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Task Queue</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSplitter::handle
{
    background-color: rgba(255, 255, 255, 80);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="taskListContainerWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="taskQueueTitleLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="logoLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../resources/logos.qrc">:/logos/logo_small.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>8</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="taskQueueTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string notr="true" extracomment="Title of a window where users can queue up multiple image geometrization tasks to be run one after another">Task Queue</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="taskListInstructionText">
              <property name="text">
               <string extracomment="Instructions to the user explaining how to add items to the task queue list, by dragging items into a list below this text">Drop image files below to batch process them via scripts</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="taskList">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="runTasksButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string extracomment="Text on a button that starts running the tasks from the task queue when pressed">Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearTaskListButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string extracomment="Text on a button that removes any queued up tasks from the task queue in the user interface">Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="taskListScriptContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="taskListScriptLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>578</width>
               <height>941</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string extracomment="The title of the File menu that contains commands relating to the handling of files, such as open, save, print, etc">File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClear_Recents">
   <property name="text">
    <string extracomment="Text on a menu item that clears a list of recently loaded files when pressed">Clear Recent Items</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string extracomment="Text on a menu item that closes the window when pressed">Exit</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="text">
    <string extracomment="Text on a menu item that opens the tutorials page when pressed">Tutorials</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string extracomment="Text on a menu item that opens the customer support/help page when pressed">Support</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string extracomment="Text on a menu item that opens the About page with information about the app when pressed">About</string>
   </property>
  </action>
  <action name="actionScript_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string extracomment="Text on a menu item that shows or hides the script console when pressed">Script Console</string>
   </property>
  </action>
  <action name="actionGlobal_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens the application-wide/global preferences page">Global Preferences</string>
   </property>
  </action>
  <action name="actionLoad_Global_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to load a file containing application-wide/global preferences page">Load Global Preferences</string>
   </property>
  </action>
  <action name="actionSave_Global_Preferences">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to save a file containing the current application-wide/global preferences page">Save Global Preferences</string>
   </property>
  </action>
  <action name="actionLoad_Scripts">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to load a file containing settings instructing the computer how to geometrize a group of queued up images">Load Task Queue Settings</string>
   </property>
  </action>
  <action name="actionSave_Scripts">
   <property name="text">
    <string extracomment="Text on a menu item that opens a dialog that allows the user to save a file containing settings instructing the computer how to geometrize a group of queued up images">Save Task Queue Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/logos.qrc"/>
 </resources>
 <connections/>
</ui>
